tests/integration/secretsmanager/test_secretsmanager.py

```diff
@@ -96,18 +96,15 @@ def _is_secret_deleted():
             )
 
     @pytest.mark.parametrize(
-        "secret_name, is_valid_partial_arn",
+        "secret_name",
         [
-            ("s-c64bdc03", True),
-            ("Valid/_+=.@-Name", True),
-            ("Valid/_+=.@-Name-a1b2", True),
-            ("Valid/_+=.@-Name-a1b2c3-", True),
-            ("Invalid/_+=.@-Name-a1b2c3", False),
```

Comment:
Removed these parameters which try to reproduce an indeterministic limitation with AWS which is no longer present in Moto.

<https://docs.aws.amazon.com/secretsmanager/latest/userguide/troubleshoot.html#ARN_secretnamehyphen>